ansible_builder/cli.py

```diff
@@ -143,6 +143,18 @@ def add_container_options(parser):
                                 ' and '.join([' for '.join([v, k]) for k, v in constants.runtime_files.items()]))
                        )
 
+        p.add_argument('--galaxy-keyring',
+                       help='Keyring for collection signature verification during installs from Galaxy. '
+                            'Will be copied into images. Verification is disabled if unset.')
+        p.add_argument('--galaxy-ignore-signature-status-code',
+                       help='A gpg status code to ignore during signature verification when installing with '
+                       'ansible-galaxy(for example, NO_PUBKEY).')
+        p.add_argument('--galaxy-required-valid-signature-count',
+                       help='The number of signatures that must successfully verify collections and rolesfrom '
```

Comment:
`roles` are not verified. There's no support for that.

I almost feel like it would be better if we didn't describe this too much and leave the galaxy documentation to fully explain. Your first sentence might be enough, plus "see docs for ansible-galaxy for further explanation" or similar.